packages/web-components/src/components.d.ts

```diff
@@ -270,10 +270,6 @@ export namespace Components {
           * The icon name
          */
         "icon": string;
-        /**
-          * The icon SVG's title attribute. Used for accessibility. If none is provided, the icon name will be used.
-         */
-        "label"?: string;
```

Comment:
Just verifying; is this change an artifact of something uncommitted from another change? cc @markacianfrani @micahjones13

Reply:
Yes, this is the title attr removal I did - https://github.com/RocketCommunicationsInc/astro/pull/628/files#